docs/Tutorials/README.md

```diff
@@ -8,27 +8,22 @@ users to learn F´ and walk through the most basic steps in developing an F´ ap
 
 ## Tutorials Index
 
-1. [HelloWorld](HelloWorld/Tutorial.md)
+1. [HelloWorld](https://github.com/fprime-community/fprime-tutorial-hello-world.git)
```

Comment:
Do you want to link to the published branded page, or directly the repository?  Same applies to all these links.